enarx-keep-sgx/src/component.rs

```diff
@@ -1,28 +1,32 @@
 // SPDX-License-Identifier: Apache-2.0
 
-use super::contents::Contents;
 use sgx_types::page::Flags;
 
 use goblin::elf::{header::*, program_header::*, Elf};
 use memory::Page;
 use span::Span;
 
 use std::cmp::{max, min};
-use std::fs::File;
-use std::io::{Read, Result};
+use std::io::Result;
 use std::ops::Range;
 use std::path::Path;
 
 /// A loadable segment of code
 pub struct Segment {
-    pub src: Contents,
+    src: Vec<Page>,
```

Comment:
Pretty sure this was the part where `pub` is removed here, but was added back again in a later commit. Will we want this to be `pub` in the end?

Reply:
Fixed.